api/test/wfl/tools/workloads.clj

```diff
@@ -20,6 +20,14 @@
 
 (def git-branch (delay (util/shell! "git" "branch" "--show-current")))
 
+(defn ^:private load-cromwell-from-env-var!
+  "Load Cromwell url from the env var ENVIRONMENT or throw."
+  []
+  (-> "ENVIRONMENT"
+      util/getenv
+      wfl/throw-or-environment-keyword!
+      stuff :cromwell :url))
```

Comment:
I guess a `def` is sufficient here ..

Reply:
Won't a def throw on compilation unless ENVIRONMENT is defined?
And what does it mean if this is nil as tested below?